pkgs/servers/web-apps/netbox/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib, pkgs, applyPatches, fetchFromGitHub, python3, plugins }:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_3;
```

Comment:
Django 3 is the default version for the foreseeable future.

Reply:
No, unfortunately not. python3Packages.django currently points to django 2.2.27

Reply:
Ah, that change is still in staging-next. https://github.com/NixOS/nixpkgs/pull/158801